app/controllers/group_assignments_controller.rb

```diff
@@ -65,6 +65,11 @@ def destroy
     end
   end
 
+  def desktop
+    code = current_user.api_token
+    redirect_to "x-github-classroom://?assignment_url=#{organization_group_assignment_url}&code=#{code}"
```

Comment:
Is it possible for a group assignment URL or a code to contain a `&` or a `+`? These characters might break the URL.